contracts/ClearingHouse.sol

```diff
@@ -14,24 +14,23 @@ import { PerpSafeCast } from "./lib/PerpSafeCast.sol";
 import { PerpMath } from "./lib/PerpMath.sol";
 import { SettlementTokenMath } from "./lib/SettlementTokenMath.sol";
 import { Funding } from "./lib/Funding.sol";
+import { AccountMarket } from "./lib/AccountMarket.sol";
+import { OpenOrder } from "./lib/OpenOrder.sol";
 import { OwnerPausable } from "./base/OwnerPausable.sol";
+import { BlockContext } from "./base/BlockContext.sol";
 import { IERC20Metadata } from "./interface/IERC20Metadata.sol";
 import { IVault } from "./interface/IVault.sol";
 import { IExchange } from "./interface/IExchange.sol";
 import { IOrderBook } from "./interface/IOrderBook.sol";
 import { IIndexPrice } from "./interface/IIndexPrice.sol";
```

Comment:
Can remove `IIndexPrice` which is unused now.

Reply:
Fixed in https://github.com/perpetual-protocol/perp-curie-contract/pull/54